cellmlmanip/parser.py

```diff
@@ -139,6 +142,27 @@ def parse(self, unit_store=None):
         # Canonicalise representation
         self.transform_constants()
 
+        # If enabled, equations are analysied for singularities in GHK equations
+        # and singularities found are fixed by inserting and appropriate piecewise
+        if not skip_singularity_fixes:
+            try:
+                # Get V
+                V = self.model.get_variable_by_ontology_term((XmlNs.OXMETA.value, 'membrane_voltage'))
+
+                # Retreive variables for which to exclude analysys to make sure their equations remain.
+                # These are: modifiable-parameters, derived quantities,
+                # annotated variables (so that they will remain findable), time (the free variable), state variables
+                tagged = set(self.model.get_variables_by_rdf((XmlNs.PYCMLMETA.value, 'modifiable-parameter'), 'yes',
```

Comment:
Can we get rid of this and replace it by an oxmeta annotation?

Reply:
I'd rather not as it would mean everybody would need to update thir cellml files.

Reply:
I've never seen this NS before though. Is it a chaste thing?

Reply:
yup

Reply:
Hmmm! I guess it doesn't rely on that namespace though, just knows to avoid variables tagged in a certain way, so it doesn't make the method less general?

Reply:
I could probably bring in these as an extra parameter I'll look at it tomorrow

Reply:
I dunno! What do @mirams and @jonc125 think?

Reply:
Yeah, it's not really oxmeta in the sense that aims to annotate biology rather than model parameters/conversion settings etc. So this was just a way to preserve a variable/quantity when importing into Chaste in particular. It's a bit legacy so happy to leave as pycml name